docs/project/2024/miruna.solomon/index.md

```diff
@@ -0,0 +1,73 @@
+# Pacman Game
+The project aims to create a Pacman game along with a wii remote
+
+:::info 
+
+**Author**: Solomon Miruna-Maria in collaboration with Balba Tudor-Neculai
```

Comment:
```suggestion
**Author**: Solomon Miruna-Maria in collaboration with Balba Tudor-Neculai \
```

Reply:
Is this a project that will have one single part that both of you will work at?

Reply:
The project has 2 parts: the game and the remote for controlling it and when we have discussed it with the laborants, they have said that it was ok for 2 students to work on it. Can we complicated it in some way in order to preserve the components we have till now? because we have order them in class with acord and maybe we can still use them in this idea

Reply:
So both of you will submit PRs?

Reply:
we were thinking yes or if it s better we can split the project in 2 parts: the game and the remote in order to be seen as parts, even if we will work both on the both sides, we haven t spllited them because we were thinking that one part requires more hardware and one is based more on soft.